fiftyone/factory/repos/delegated_operation.py

```diff
@@ -189,6 +189,17 @@ def _create_indexes(self):
                 )
             )
 
+        if "group_id_1_status_1" not in index_names:
+            indices_to_create.append(
+                IndexModel(
+                    [
+                        ("group_id", pymongo.ASCENDING),
+                        ("status", pymongo.ASCENDING),
```

Comment:
looks like it's `run_state` not `status`, that's my fault. Idk what `status` is because it exists on my local docs but is unset 🤷🏼